native/src/modules/endpoint/sagas/__tests__/watchFetchCategory.spec.js

```diff
@@ -9,22 +9,38 @@ import { expectSaga, testSaga } from 'redux-saga-test-plan'
 import loadCityContent from '../loadCityContent'
 import CategoriesMapModelBuilder from '../../../../testing/builder/CategoriesMapModelBuilder'
 import ErrorCodes from '../../../error/ErrorCodes'
+import moment from 'moment'
 
 jest.mock('rn-fetch-blob')
 jest.mock('../loadCityContent')
+// $FlowFixMe Date.now is writable here
+Date.now = jest.fn().mockReturnValue(new Date('2020-01-01T12:00:00.000Z'))
 
 const createDataContainer = async (city: string, language: string) => {
   const categoriesBuilder = new CategoriesMapModelBuilder(city, language)
   const categories = categoriesBuilder.build()
   const resources = categoriesBuilder.buildResources()
   const languages = new LanguageModelBuilder(2).build()
+  const metaCities = {
+    [city]: {
+      languages: { [language]: { lastUpdate: moment('2020-01-01T00:00:00.000Z') } },
+      lastUsage: moment('2020-01-01T01:00:00.000Z')
+    }
+  }
 
   const dataContainer = new DefaultDataContainer()
   await dataContainer.setCategoriesMap(city, language, categories)
   await dataContainer.setLanguages(city, languages)
   await dataContainer.setResourceCache(city, language, resources)
+  await dataContainer._databaseConnector._storeMetaCities(metaCities)
```

Comment:
Tests should not access internals. Is there a way to test this differently or remove the _ in the function names?